src/EditorFeatures/Core.Wpf/Suggestions/AsyncSuggestedActionsSource.cs

```diff
@@ -19,12 +20,14 @@
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
 using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Shared.Collections;
+using Microsoft.CodeAnalysis.PooledObjects;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
 {
     internal partial class SuggestedActionsSourceProvider
```

Comment:
`SuggestedActionsSourceProvider` needs to be decorated with `SuggestedActionPriorityAttribute` which determines the priorities you support (e.g. `DefaultOrderings.High` and `DefaultOrderings.Default`)